kinto_http/__init__.py

```diff
@@ -655,6 +655,27 @@ def get_records(self, *, collection=None, bucket=None, **kwargs):
                                      collection=collection)
         return self._paginated(endpoint, **kwargs)
 
+    def get_paginated_records(self, *, collection=None, bucket=None, **kwargs):
+        endpoint = self.get_endpoint('records',
+                                     bucket=bucket,
+                                     collection=collection)
+
+        yield from self._paginated_gen(endpoint, **kwargs)
+
+    def _paginated_gen(self, endpoint, records=None, *, if_none_match=None, **kwargs):
```

Comment:
Can you explain the parameter records here?

Reply:
There is no need for   `records` here, I had them because I used the `_paginated` method signature for the  `_paginated_gen`. I'll remove that one.